assignments/closure.js

```diff
@@ -1,21 +1,26 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+const sayHi = 'Hi Asha!';
 
+function returnsayHi() {
+    return sayHi;
+}
+returnsayHi();
```

Comment:
this is a closure in the very basic sense of the word, because the function is reaching outside it's scope and storing it's value inside it's execution context.  Usually, when we return a nested function inside another function, we see a better use case for closures.